datafusion/proto/src/physical_plan/to_proto.rs

```diff
@@ -330,6 +332,17 @@ impl TryFrom<Arc<dyn PhysicalExpr>> for protobuf::PhysicalExprNode {
                     ),
                 ),
             })
+        } else if let Some(expr) = expr.downcast_ref::<LikeExpr>() {
```

Comment:
I wonder if there is any test coverage for round tripping a physical `LikeExpr`?